getting_started/introduction/godot_design_philosophy.rst

```diff
@@ -98,10 +98,11 @@ game code. See `The Godot editor is a Godot game`_ below.
 Open source
 -----------
 
-Godot offers a fully open source codebase under the **MIT license**.
-This means all the technologies that ship with it have to be Free
-(as in freedom) as well.
-For the most part, they're developed from the ground up by contributors.
+Godot offers a fully open source codebase under the **MIT license**. This means
+that the codebase is free for anyone to download, use, modify, or share, as long
+as its license file is kept intact. Thanks to this open source license, most of
+the technologies that define Godot are developed from the ground up by
+community contributors.
```

Comment:
Some meaning from the original text was lost. Specifically, it's not just that we can thank the open source license for enabling contributors to develop things from the ground up, but rather it forces this situation in cases where existing solutions are proprietary or under restrictive FOSS licenses. Feel free to tweak this.

```suggestion
Godot offers a fully open source codebase under the **MIT license**.
This means that the codebase is free for anyone to download, use,
modify, or share, as long as its license file is kept intact.

All technologies that ship with Godot, including third-party libraries, must
be legally compatible with this open source license. Therefore, most parts
of Godot are developed from the ground up by community contributors.
```

Reply:
LGTM!